apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -876,6 +881,16 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
     }
 
 
+def get_default_partial_credit_type(grading_method):
+    # For backward compatibility, we need to override the default partial credit type
+    # when grading_method = ORDERED
+    return (
+        PartialCreditType.NONE
+        if grading_method is GradingMethodType.ORDERED
+        else PartialCreditType.LCS
+    )
```

Comment:
having this as a function is nice. You should at the input and output type declarations too.

Reply:
Yeah, this was used both in `grade()` and in `test()`, so I figured it'll be a good idea. Added type annotation in [35cc326](https://github.com/PrairieLearn/PrairieLearn/pull/8151/commits/35cc326bca7234eca18e3687711275fe8a4a04eb).